src/flag_gems/ops/upsample_bicubic2d_aa.py

```diff
@@ -378,7 +366,10 @@ def upsample_bicubic2d_aa_kernel(
 
 
 # upsample and downsample
-@triton.autotune(configs=configs(), key=["N", "C", "OH", "OW"])
+@triton.autotune(
+    configs=runtime.get_triton_config("upsample_bicubic2d_aa"),
+    key=["N", "C", "OH", "OW"],
+)
 @triton.jit
 def general_interpolate_bicubic2d_aa_kernel(
```

Comment:
suggest add device guard for it incidentally.

Reply:
Done